service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -357,6 +359,20 @@ private void sendOrderNotificationForLocalAuthority(final CaseDetails caseDetail
             Long.toString(caseDetails.getId()));
     }
 
+    private String getAdminEmail(EventData eventData) {
+        String ctscValue = getCtscValue(eventData.getCaseDetails().getData());
+
+        if (ctscValue.equals("Yes")) {
+            return ctscEmailLookupConfiguration.getEmail();
+        }
+
+        return hmctsCourtLookupConfiguration.getCourt(eventData.getLocalAuthorityCode()).getEmail();
+    }
+
+    private String getCtscValue(Map<String, Object> caseData) {
+        return caseData.get("sendToCtsc") != null ? caseData.get("sendToCtsc").toString() : "No";
```

Comment:
would this value ever be null?

Reply:
Probably not as it's set in 'about to submit'. Notifications then happen in the submitted. I guess it's just added for additional safety.